R/navbar.R

```diff
@@ -159,6 +162,68 @@ navbar_articles <- function(pkg = ".") {
   print_yaml(menu)
 }
 
+navbar_brand <- function(pkg, depth) {
+  bs_version <- pkg$bs_version
+  meta_navbar_brand <- pkg$meta$navbar$brand
+  components <- modify_list(brand_components(pkg, depth), meta_navbar_brand$components)
+  structure <- meta_navbar_brand$structure %||% brand_structure()
+  paste0(components[structure], collapse = " ")
+}
+
+brand_structure <- function() {
+   c("package", "version")
+}
+
+brand_components <- function(pkg, depth) {
+  bs_version <- pkg$bs_version
+  list(
+    package = navbar_package_brand(pkg, bs_version, depth),
+    version = navbar_package_version(pkg, bs_version)
+  )
+}
+
+navbar_package_brand <- function(pkg, bs_version, depth) {
+
+  root <- up_path(depth)
+  title <- pkgdown_site_title(pkg)
+
+  if (bs_version == 3) {
```

Comment:
In the interests of simplicity, maybe we should make this only apply to BS5 sites?

Reply:
Maybe, but wouldn't it make the docs more complicated? (that was my motivation :sweat_smile: )

Reply:
I don't thinks, because the customisation docs already only apply to BS5 sites.